### DISCUSSION TITLE
Handle sign ups with Supabase' Auth UI component

### DISCUSSION THREAD

**Discussion by FacuBozzi** [supabase/supabase#15065] [Questions]
Hello. Im using the Auth UI component in my next.js app and even though the magic link and integration is amazing I can't seem to find a way to handle user signup/login.

I want the redirect to be to different URLs depending if the user signed up or just logged in. The reason I need this is because when a user just signed up for the first time (either from email or with a social provider like Google) I need the redirect to be to some url like "/last-steps" where they finish configurating their account. Is this possible with Supabase? Other workarounds are welcomed.

Thanks in advance!

**Comment by GaryAustin1** [comment]
There is nothing built in to distinguish between new and existing user.  The easiest way is to have a profile table or use user or app metadata columns to have a flag or just check if the follow up data exists when signin occurs.
